Parse/src/main/java/com/parse/ParseRESTCommand.java

```diff
@@ -189,7 +191,7 @@ public static ParseRESTCommand fromJSONObject(JSONObject jsonObject) {
   private static String createUrl(String httpPath) {
     // We send all parameters for GET/HEAD/DELETE requests in a post body,
     // so no need to worry about query parameters here.
-    return String.format("%s/1/%s", ParseObject.server, httpPath);
+    return String.format("%s/%s", server, httpPath);
```

Comment:
Can we use url building here, to avoid problems with `/` at the end of the server.